### ISSUE TITLE
[Question] Do we have plan to support Terraform

### ISSUE THREAD

**Issue by WestFarmer** [OWASP/Nettacker#893]
Do we have plan to support Terraform ?

make terraform provider plugin of this tool

**Comment by securestep9** [comment]
yes we have this in the long-range plans, but we need volunteers who can help the project and spare some time developing the plugin - are you willing to contribute ?

**Comment by bhagyashree980** [comment]
Hii @securestep9 ,I wanted to express my interest in contributing to the DevOps project you’re working on. I’m excited about the opportunity to be part of this initiative and am ready to spare time and help develop the plugin.
Please let me know how I can get started or if there are any specific tasks
